openfga/v2beta1/openfga.proto

```diff
@@ -0,0 +1,157 @@
+syntax = "proto3";
+
+package openfga.v2beta1;
+
+import "google/api/field_behavior.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "validate/validate.proto";
+
+message Object {
+  string type = 1;
+  string id = 2;
+}
+
+// User represents either a concrete object or a set of users (userset).
+message User {
+  oneof ref {
+    Object user = 1;
```

Comment:
To avoid the ambiguous `user` term here, should we just change this to `object`? It makes more sense to me that a User can be a concrete object or a userset.

Thoughts?

Reply:
why not change `message user` to `message subject`? or is that a can of worms? :P